src/main/java/core/basesyntax/FigureAction.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface FigureAction {
```

Comment:
let's create two separate interface one for draw() -> Drawable and second one for getArea() -> AreaCalculator